build/esbuild-watch.ts

```diff
@@ -0,0 +1,11 @@
+import esbuild from "esbuild";
+import { esBuildContext } from "./esbuild-build";
+
+async function watch() {
+  const ctx = await esbuild.context(esBuildContext);
+  await ctx.watch();
+  console.log("Watching...");
+}
+
+// This MUST NOT be awaited.
+void watch();
```

Comment:
```suggestion
void watch().catch(console.error);
```

Probably can catch errors like this